package.json

```diff
@@ -13,21 +13,8 @@
     "build-all": "tsc -b"
   },
   "devDependencies": {
-    "@types/jest": "^24.0.22",
-    "@types/node": "^10.17.5",
-    "@types/nodeunit": "^0.0.30",
-    "conventional-changelog-cli": "^2.0.27",
-    "fs-extra": "^8.1.0",
-    "jest": "^24.9.0",
-    "jsii-diff": "^0.20.3",
-    "jsii-pacmak": "^0.20.3",
     "lerna": "^3.18.4",
-    "nodeunit": "^0.11.3",
-    "nyc": "^14.1.1",
-    "standard-version": "^7.0.0",
-    "ts-jest": "^24.1.0",
-    "tslint": "^5.20.1",
-    "typescript": "~3.6.4"
```

Comment:
ha?

Reply:
Dropped LOTS of un-needed dependencies (made install slower, introduced risk of taking "shadow" dependencies on those, etc...)